incubator/elasticsearch/values.yaml

```diff
@@ -120,5 +120,25 @@ data:
   updateStrategy:
     type: OnDelete
 
+prometheusExporter:
+  # The values within here are responsible for setting up service ports and sidecar exporter containers.
+  repository: justwatch/elasticsearch_exporter
+  tag: 1.0.2
+  pullPolicy: "IfNotPresent"
+  clientName: client-exporter
+  internalPort: 9108
+  externalPort: 9108
+  metricsEndpoint: /metrics
+
+serviceMonitor:
```

Comment:
This option could benefit from a comment linking it to prometheus-operator / kube-prometheus